### PR TITLE
Remove invalid import from Watson NLU tests

### PR DESCRIPTION
**Pull request by BryanCutler**

### CODE REVIEW

**Comment by BryanCutler** [comment]
cc @frreiss , noticed this wasn't enabled in testing before and there as an import issue. I will make the release after this is merged.

File: text_extensions_for_pandas/io/watson/test_nlu.py [hunk 1]
```diff
@@ -424,8 +424,9 @@ def setUpClass(cls):
     def _make_request():
         from ibm_watson import NaturalLanguageUnderstandingV1
         from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
-        from ibm_watson.natural_language_understanding_v1 import Features, CategoriesOptions, ConceptsOptions, EmotionOptions, EntitiesOptions, KeywordsOptions, \
-            MetadataOptions, RelationsOptions, SemanticRolesOptions, SentimentOptions, SyntaxOptions, SyntaxOptionsTokens
```
Comment (BryanCutler) [inline text_extensions_for_pandas/io/watson/test_nlu.py:428]: `MetadataOptions` was invalid, must have been from a very old version.

Comment (BryanCutler) [reply text_extensions_for_pandas/io/watson/test_nlu.py:428]: Looks like it has been renamed to from `FeaturesResultsMetadata`, I'll update the comment but this is not currently being used

Comment (BryanCutler) [reply text_extensions_for_pandas/io/watson/test_nlu.py:428]: The API reference still lists it as valid, I'll open an issue
https://cloud.ibm.com/apidocs/natural-language-understanding?code=python#metadata

**Review by frreiss — APPROVED** [review]
LGTM

File: text_extensions_for_pandas/io/watson/test_nlu.py (hunk 1 above)
Comment (BryanCutler) [reply text_extensions_for_pandas/io/watson/test_nlu.py:428]: Made issue https://github.com/watson-developer-cloud/python-sdk/issues/778

**Pull request merged**